app/views/foreman_discovery/debian_kexec.erb

```diff
@@ -0,0 +1,36 @@
+<%#
+kind: kexec
+name: Debian kexec
+oses:
+oses:
+- Debian 6.0
+- Debian 7.
```

Comment:
in the community-templates, we did cut that down to just Debian and Ubuntu to let all versions associate